.changeset/small-grapes-cross.md

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
some of these packages were still on v17, @emmenko 
Shall we generate a "final" changeset, setting all packages to be v19?
what do you think?

Reply:
Sure, we can do that.